Blood and Gold/sheet.json

```diff
@@ -0,0 +1,8 @@
+{
+  "html": "bloodandgold.html",
+  "css": "bloodandgold.css",
+  "authors": "Rodney Nedlose",
+  "preview": "Blood and Gold.PNG",
```

Comment:
This needs to read `"preview": "Blood and Gold.png",

Reply:
All done.  Thanks!